packages/core-components/src/overridableComponents.ts

```diff
@@ -157,12 +164,15 @@ type BackstageComponentsNameToClassKey = {
   BackstageSidebarSpacer: SidebarSpacerClassKey;
   BackstageSidebarDivider: SidebarDividerClassKey;
   BackstageSidebarItem: SidebarItemClassKey;
+  BackstageSidebarSubmenu: SidebarSubmenuClassKey;
+  BackstageSidebarSubmenuItem: SidebarSubmenuItemClassKey;
   BackstageSidebarPage: SidebarPageClassKey;
   BackstageCustomProvider: CustomProviderClassKey;
   BackstageSignInPage: SignInPageClassKey;
   BackstageTabbedCard: TabbedCardClassKey;
   BackstageTabbedCardBoldHeader: BoldHeaderClassKey;
   BackstageCardTab: CardTabClassKey;
+  Link: LinkClassKey;
```

Comment:
We'll want to prefix these in the same way as the rest to whatever extent that's possible. Let's worry less about the RHS name, i.e. `LinkClassKey` in this case

Reply:
Took me forever to figure out what RHS meant: right hand side, lol. I've fixed this as well as all the others in this PR.